### PR TITLE
chore: well component to card component(Updating PR #12200)

### PR DESCRIPTION
**Pull request by yardz**
### SUMMARY
This PR is a rebase of PR https://github.com/apache/superset/pull/12200.

I downloaded the branch with PR, did a rebase with `apache/master`, then resolved the conflicts. After finishing the rabase, some errors appeared (such as unused imports).

Resolvi os erros de lint e então fiz o commit nessa nova branch.

### CODE REVIEW

**Comment by junlincc** [comment]
@yardz Thank you, Bruno, for all your great work. 🙏

File: superset-frontend/src/common/components/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { styled } from '@superset-ui/core';
+// eslint-disable-next-line no-restricted-imports
```
Comment (michael-s-molina) [inline superset-frontend/src/common/components/Card.tsx:21]:
```suggestion
```

File: superset-frontend/src/common/components/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import { Card as AntdCard } from 'antd';
+import { CardProps as AntdCardProps } from 'antd/lib/card';
```
Comment (michael-s-molina) [inline superset-frontend/src/common/components/Card.tsx:23]:
```suggestion
import AntdCard, { CardProps as AntdCardProps } from 'antd/lib/card';
```

File: superset-frontend/src/common/components/common.stories.tsx [hunk 3]
```diff
@@ -206,6 +209,76 @@ export const Switch = () => (
...
+export const BadgeColored = () => <Badge color="blue" text="Blue" />;
+export const BadgeTextColored = () => (
+  <Badge textColor="yellow" color="red" text="yellow" />
+);
+export const BadgeSuccess = () => <Badge status="success" text="Success" />;
+export const BadgeError = () => <Badge status="error" text="Error" />;
+export const BadgeSmall = () => <Badge count={100} size="small" />;
+
+export const CardDefault = () => <Card />;
+export const CardPadded = () => <Card padded />;
```
Comment (michael-s-molina) [inline superset-frontend/src/common/components/common.stories.tsx:222]:
```suggestion
```
```suggestion
export const BadgeDefault = () => <Badge count={100} />;
export const BadgeColored = () => <Badge color="blue" text="Blue" />;
export const BadgeTextColored = () => (
  <Badge textColor="yellow" color="red" text="yellow" />
);
export const BadgeSuccess = () => <Badge status="success" text="Success" />;
export const BadgeError = () => <Badge status="error" text="Error" />;
export const BadgeSmall = () => <Badge count={100} size="small" />;

export const CardDefault = () => <Card />;
export const CardPadded = () => <Card padded />;
```

File: superset-frontend/src/common/components/common.stories.tsx [hunk 4]
```diff
@@ -206,6 +209,76 @@ export const Switch = () => (
...
+export const CollapseGhost = () => (
+  <Collapse defaultActiveKey={['1']} ghost>
+    <Collapse.Panel header="Hi! I am a header" key="1">
+      Hi! I am a sample content
+    </Collapse.Panel>
+    <Collapse.Panel header="Hi! I am another header" key="2">
+      Hi! I am another sample content
+    </Collapse.Panel>
+  </Collapse>
+);
```
Comment (michael-s-molina) [inline superset-frontend/src/common/components/common.stories.tsx:243]:
```suggestion
```

File: superset-frontend/src/common/components/common.stories.tsx [hunk 5]
```diff
@@ -206,6 +209,76 @@ export const Switch = () => (
...
+    </Collapse.Panel>
+    <Collapse.Panel
+      header="Hi! I am another header"
+      key="2"
+      style={{ background: '#BBB' }}
+    >
+      Hi! I am another sample content
+    </Collapse.Panel>
+  </Collapse>
+);
```
Comment (michael-s-molina) [inline superset-frontend/src/common/components/common.stories.tsx:281]:
```suggestion
```

File: superset-frontend/src/common/components/common.stories.tsx [hunk 6]
```diff
@@ -20,13 +20,16 @@ import React, { useState, useRef, useCallback } from 'react';
 import { action } from '@storybook/addon-actions';
 import { withKnobs, boolean, select } from '@storybook/addon-knobs';
 import Button from 'src/components/Button';
+import Badge from 'src/components/Badge';
```
Comment (michael-s-molina) [inline superset-frontend/src/common/components/common.stories.tsx:23]:
```suggestion
```

File: superset-frontend/src/common/components/common.stories.tsx [hunk 7]
```diff
@@ -20,13 +20,16 @@ import React, { useState, useRef, useCallback } from 'react';
...
 import Modal from './Modal';
 import Tabs, { EditableTabs } from './Tabs';
 import { Tooltip as AntdTooltip } from './Tooltip';
 import { Switch as AntdSwitch } from './Switch';
 import { Menu, Input, Divider } from '.';
 import { Dropdown } from './Dropdown';
 import InfoTooltip from './InfoTooltip';
+import Card from './Card';
+import Collapse from './Collapse';
```
Comment (michael-s-molina) [inline superset-frontend/src/common/components/common.stories.tsx:32]:
```suggestion
```

File: superset-frontend/src/common/components/common.stories.tsx (hunk 3 above)
Comment (michael-s-molina) [reply superset-frontend/src/common/components/common.stories.tsx:222]: These components already have stories in their folders.

**Comment by michael-s-molina** [comment]
@yardz Thanks for rebasing this PR! 🚀

**Review by pkdotson — APPROVED** [review]
LGTG after the requested changes!

**Pull request merged**